core/src/main/kotlin/org/lflang/generator/rust/RustModel.kt

```diff
@@ -715,7 +726,7 @@ private val TypeParm.identifier: String
 val BuildType.cargoProfileName: String
     get() = when (this) {
         BuildType.DEBUG             -> "debug"
-        BuildType.TEST              -> "test"
+        BuildType.TEST              -> "lf-test"
```

Comment:
@oowekyala, I'm unsure where this changes comes from. Does it look right to you?